packages/docs/docs/concepts/cornerstone-core/renderingEngine.md

```diff
@@ -30,6 +30,78 @@ canvas get updated, and at render time, we copy from offscreen to onscreen for e
 
 For instance for PET-CT fusion which has 3x3 layout which includes CT (Axial, Sagittal, Coronal), PET (Axial, Sagittal, Coronal) and Fusion (Axial, Sagittal, Coronal), we create two volume mappers for CT and PET individually, and for the Fusion viewports we re-use both created textures instead of re-creating a new one.
 
+## Rendering Engine Implementations
+
+Cornerstone3D provides two rendering engine implementations to handle different use cases and overcome technical limitations:
+
+### StandardRenderingEngine
+
+The `StandardRenderingEngine` is the original implementation that uses a single, large offscreen canvas for all viewports. This approach:
+
+- Creates one massive offscreen canvas that grows horizontally as viewports are added
+- Renders all viewports to specific coordinates on this single offscreen canvas
+- Copies pixel data from the offscreen canvas to individual onscreen viewports
+
+**Limitations of StandardRenderingEngine:**
+
+- **Canvas Size Limits**: Browsers impose maximum canvas dimensions (e.g., 16,384px in Chrome). When the combined width of all viewports exceeds this limit, the offscreen canvas is silently cropped, causing severe visual artifacts, misaligned viewports, and blank viewports
+- **Performance Degradation**: As the offscreen canvas approaches size limits, performance degrades significantly, especially on high-resolution displays or layouts with many viewports
+- **Multi-Monitor Issues**: Practically impossible to use across multiple high-resolution monitors due to canvas size limitations
+- **Memory Consumption**: Allocates a huge, memory-intensive offscreen canvas regardless of actual viewport usage
+
+### NextRenderingEngine (SequentialRenderingEngine)
```

Comment:
I think we just say sequentialRE, not next, since 2 years from now another thing is next